src/main/resources/reference.conf

```diff
@@ -514,11 +514,14 @@ resourceTypes = {
       read_policies = {
         description = "view all policies and policy details for this workflow-collection"
       }
+      list_children = {
+        description = "list child resources"
+      }
     }
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["delete", "add", "view", "abort", "get_cost", "update", "alter_policies", "read_policies"]
+        roleActions = ["delete", "add", "view", "abort", "get_cost", "update", "alter_policies", "read_policies", "list_children"]
```

Comment:
I'm having second thoughts on adding this action to the `workflow-collection` resource type. None of the roles currently allow users to set children on a `workflow-collection`, so I don't know that we really need the ability to list out children since there can't be any

Reply:
Is it just that the ticket was being overly cautious, then?
I'll remove requirement to check the workflow-collection from the ticket as well

Reply:
Yeah, I think I included the note on the `workflow-collection` because if it did actually have a child then that would cause issues, but that feels overly cautious after seeing the existing role actions